steppers/ldo/42sth40-1684ac/2130/24v-0.4a-extruder.cfg

```diff
@@ -0,0 +1,8 @@
+[tmc2130 extruder]
+sense_resistor: 0.22
+run_current: 0.52
+hold_current: 0.2
```

Comment:
Delete this (see: https://www.klipper3d.org/TMC_Drivers.html#prefer-to-not-specify-a-hold_current)

Reply:
Done